tests/rbd/test_rbd_group_commands_validation.py

```diff
@@ -0,0 +1,329 @@
+"""Module to validate rbd group-related commands.
+
+Test case covered -
+CEPH-83594545 - RBD Group-related commands validation
+
+Pre-requisites :
+1. Cluster must be up and running with capacity to create pool
+2. We need atleast one client node with ceph-common package,
+   conf and keyring files
+
+Test Case Flow:
+1. Deploy Ceph on version 8.0 or greater.
+2. Create an RBD pool named pool1 using ceph osd pool create pool1
+3. Create an RBD group named group1 in pool1 using rbd group create pool1/group1
+4. Create an RBD image named image1 in pool1 using rbd create pool1/image1 --size 1024
+5. Add the image image1 to the group group1 using rbd group image add pool1/group1 pool1/image1
+6. List images in the group group1 using rbd group image list pool1/group1
+7. Remove the image image1 from the group group1 using rbd group image remove pool1/group1 pool1/image1
+8. List the RBD groups in pool1 using rbd group list pool1
+9. Rename the group group1 to group2 within pool1 using rbd group rename pool1/group1 pool1/group2
+10. Create a snapshot of the group group2 named snap1 using rbd group snap create pool1/group2 --snap snap1
+11. List snapshots of the group group2 using rbd group snap list pool1/group2
+12. Remove the snapshot snap1 from the group group2 using rbd group snap remove pool1/group2 --snap snap1
+13. Rename the group's snapshot snap1 to snap2 using rbd group snap rename pool1/group2 --snap snap1 <new-snap-name>
+14. Rollback the group group2 to the snapshot snap2 using rbd group snap rollback pool1/group2 --snap snap2
+15. Delete the group group2 using rbd group remove pool1/group2
+"""
+
+from copy import deepcopy
+
+from ceph.rbd.initial_config import initial_rbd_config
+from ceph.rbd.utils import getdict, random_string
+from ceph.rbd.workflows.cleanup import cleanup
+from ceph.utils import get_node_by_id
+from cli.rbd.rbd import Rbd
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def validate_rbd_group_commands(rbd_obj, client, **kw):
+    """
+        validate rbd group-related commands.
+    Args:
+        rbd_obj: RBD object
+        client : client node object
+        **kw: any other arguments
+    """
+
+    kw["client"] = client
+    rbd = rbd_obj.get("rbd")
+
+    for pool_type in rbd_obj.get("pool_types"):
+        rbd_config = kw.get("config", {}).get(pool_type, {})
+        multi_pool_config = deepcopy(getdict(rbd_config))
+
+        for pool, pool_config in multi_pool_config.items():
+            if "data_pool" in pool_config.keys():
+                _ = pool_config.pop("data_pool")
+
+            # Create an RBD group in pool
+            group = "group_" + random_string(len=4)
+            out, err = rbd.group.create(**{"group-spec": f"{pool}/{group}"})
+            if err:
+                log.error(f"Create group {group} failed with error {err}")
+                return 1
+            else:
+                log.info(f"Successfully created pool {pool}/{group}")
```

Comment:
`It should be Group created successfully under pool {pool}/{group}`

Reply:
Addressed